installer/README.md

```diff
@@ -31,9 +31,9 @@ docker run -it --net=host -v ~/go/src/github.com/vmware/vic/bin:/test-bin \
   gcr.io/eminent-nation-87317/vic-integration-test:1.22 ovftool --acceptAllEulas --X:injectOvfEnv \
   --X:enableHiddenProperties -st=OVA --powerOn --noSSLVerify=true -ds=datastore1 -dm=thin \
   --net:Network="VM Network" --prop:appliance.email_from="noreply@vmware.com" \
-  --prop:appliance.root_pwd="VMware1" --prop:appliance.permit_root_login=true --prop:harbor.port=443 \
-  --prop:admiral.port=8282 --prop:harbor.admin_password="VMware1\!" \
-  --prop:harbor.db_password="VMware1\!" /test-bin/vic-1.1.0-a84985b.ova \
+  --prop:appliance.root_pwd="VMware1\!" --prop:appliance.permit_root_login=True --prop:registry.port=443 \
+  --prop:management_portal.port=8282 --prop:registry.admin_password="VMware1\!" \
+  --prop:registry.db_password="VMware1\!" /test-bin/vic-1.1.0-a84985b.ova \
```

Comment:
Put those passwords between single quotes `'` and remove escape chars.

(`--prop:appliance.root_pwd='VMware1!'` instead of `--prop:appliance.root_pwd="VMware1\!"`)